pkg/container/logger/logger.go

```diff
@@ -29,28 +30,68 @@ type LogsRedis struct {
 	Stderr string `json:"stderr"`
 }
 
+type marshaler interface {
```

Comment:
I can see you have good intentions to cover everything with tests but this can very easily go out of hand. 

If you think about it this test make the code unmanageable after a while and you have to choose what need to be tested and what not.

For example if you think about the value of this test it's very very small because there is not actual logic here to test. 

The Serialize function all what it does is it serialize using a standard json serializer and then write the content to a file using a write file from the standard library. 

So what does the test cover? the test code is already more than the logic because there is no logic in this function, no? and we definitely don't want to test json serializer nor the os.WriteFile functions.

Please rever this file and let's discuss what can be (and should) be tested and what's not.

Also please skip the entire `pkg/container` because it's not supported for workloads anymore it's only here because we use it internally to run zdb.